client/cypress/e2e/activities-spec.cy.js

```diff
@@ -0,0 +1,37 @@
+describe('Activities page', () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'http://localhost:3001/api/v1/entries', {
+      statusCode: 201,
+      fixture: 'entry.json'
+    })
+    cy.intercept('GET', 'http://localhost:3001/api/v1/entries', {
+      statusCode: 200,
+      fixture: 'entries.json'
+    })
+    cy.visit('http://localhost:3000/what-should-you-do/Anxious')
```

Comment:
This is seriously brilliant KB! I would have never thought to do this. Great idea & I'm definitely stealing this for future testing!!